Card/build.gradle

```
@@ -1,6 +1,7 @@
 plugins {
     id 'com.android.library'
     id 'kotlin-android'
+    id("io.gitlab.arturbosch.detekt").version("1.18.0-RC2")
```

```suggestion
    id 'io.gitlab.arturbosch.detekt' version '1.18.0-RC2'
```
Seems more in line with the [Gradle site's style](https://docs.gradle.org/current/userguide/plugins.html#sec:plugins_block).

---

Good call! Updated this in the last commit.